src/views/Leads/index.js

```diff
@@ -172,7 +192,7 @@ export default class Leads extends React.PureComponent {
                     assignee ? (
                         <Link
                             key={assignee}
-                            className={styles.assigneeLink}
+                            className="assignee-link"
```

Comment:
Why is a non-scoped className used here?

Reply:
The styles for Tables are stored in separate file (inside Tables/styles.scss) so non-scoped className.